common/app/model/LiveBlogCurrentPage.scala

```diff
@@ -247,9 +247,9 @@ sealed trait PageReference {
 
   def isArchivePage: Boolean
 
-  def buildQueryParam(topMentionResult: Option[TopMentionsResult]) = {
-    topMentionResult match {
-      case Some(value) => s"&topics=${topMentionResult.get.`type`}:${value.name}"
+  def buildQueryParam(topicResult: Option[TopicResult]) = {
+    topicResult match {
+      case Some(value) => s"&topics=${topicResult.get.`type`}:${value.name}"
```

Comment:
This was my mistake previously, but now that you're changing it, could you please use value.`type` instead of topicResult.get.`type`